dir.targets

```diff
@@ -52,15 +57,21 @@
 
     <!-- Restore build tools -->
     <Exec Command="$(NugetRestoreCommand) &quot;$(SourceDir).nuget\packages.config&quot;" StandardOutputImportance="Low" />
-    
-    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true'"
-           Text="The build tools package was just restored and so we cannot continue the build of an individual project because targets from the build tools package were not able to be imported. Please retry the build the individual project again." />
+
+    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true' and '$(BuildingInsideVisualStudio)'=='true'"
+      Text="The build can not be completed as the build tools package was just restored. Please reload the solution to allow targets from the build tools package to be imported." />
```

Comment:
OK. I think I understand better after playing with this and I believe it should be a warning, not an error in the VS case. AFAICT, IntelliSense is degraded until you reload, but the actual out-of-proc build process picks up the targets just fine. I can see all of the buildtools w/ verbosity=normal) no matter what I do. If I'm right, I don't think it makes sense to abort a build that would have worked perfectly just to alert me to reload so I can get better design-time behaviour.